src/main/scala/org/hammerlab/guacamole/assembly/DeBruijnGraph.scala

```diff
@@ -123,13 +125,13 @@ class DeBruijnGraph(val kmerSize: Int,
   }
 
   /**
-   *  Searches forward or backward from a node to find those connected by a unique path
+    *  Searches forward or backward from a node to find those connected by a unique path
```

Comment:
I'll leave it as-is now (which should be first asterisk). I'm indifferent on the issue otherwise. As with the other style things, probably something else (intellij/scalariform) should worry about them rather than us, so I'll set my settings appropriately after #469